README.md

```diff
@@ -25,11 +25,12 @@ poetry install
 ### Initialization
 To start, initialize the drsclient. There are only two fields required to initialize the client:
 - `baseurl` (required): url of the server we're trying to connect with.
-- `auth` (optional): auth info to access object bytes.
+- `auth` (optional): auth info to access object bytes or POST/DELETE bundles in Gen3 DRS servers.
+- `token` (optional): access token to access object bytes or POST/DELETE bundles in Gen3 DRS servers.
 ```python
 from drsclient.client import DrsClient
 
-client = DrsClient(baseurl="somedrsserver.io", auth="some_token")
+client = DrsClient(baseurl="somedrsserver.io", auth=(username, password), token="some token")
```

Comment:
i would separate the use of `auth` and the use of `token` into 2 example lines, to make it clear you're not supposed to use both

also, need to document that `auth` takes priority over `token` if using both